client/lib/screen-title/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { setTitle as legacySetTitle } from './actions';
+
+/**
+ * Given a Redux store instance, subscribes to the store, updating the page
+ * when the title state changes. Currently, this dispatches through the legacy
+ * Flux actions used elsewhere in the codebase.
+ *
+ * @param {Object} store Redux store instance
+ */
+export function subscribeToStore( store ) {
+	let title = store.getState().documentSettings.title;
```

Comment:
Since `documentSettings` is not wired in to `client/state/index.js` yet, this will result in an error.